src/lv_widgets/lv_msgbox.c

```diff
@@ -281,7 +281,7 @@ void lv_msgbox_stop_auto_close(lv_obj_t * mbox)
 
 /**
  * Set whether recoloring is enabled
- * @param btnm pointer to button matrix object
+ * @param mbox pointer to button matrix object
```

Comment:
Can you also change the wording to say "message box" and not "button matrix", please?

Reply:
Thanks, will fix it asap!